pkg/Cpanel/Security/Advisor/Assessors/Imunify360.pm

```diff
@@ -77,25 +75,33 @@ sub generate_advice {
         }
 
         # These checks will only run on v88 and highger.
-        if (   Cpanel::Version::compare( $cpanel_version, '>=', $IMUNIFYAV_MINIMUM_CPWHM_VERSION )
-            && require Whostmgr::Store::Product::ImunifyAVPlus
+        if ( Cpanel::Version::compare( $cpanel_version, '>=', $IMUNIFYAV_MINIMUM_CPWHM_VERSION )
             && ( !$self->{i360}{installed} || !$self->{i360}{licensed} ) ) {
 
-            my $store = Whostmgr::Store::Product::ImunifyAVPlus->new( redirect_path => 'cgi/securityadvisor/index.cgi' );
+            if ( _can_load_module('Whostmgr::Store::Product::ImunifyAV') ) {
+                my $iav_store = Whostmgr::Store::Product::ImunifyAV->new( redirect_path => 'cgi/securityadvisor/index.cgi' );
+                $self->{iav}{installed} = $iav_store->is_product_installed();
+                $self->_suggest_iav;
+            }
+
+            if ( _can_load_module('Whostmgr::Store::Product::ImunifyAVPlus') ) {
 
-            if ( $store->should_offer() ) {
+                my $iavp_store = Whostmgr::Store::Product::ImunifyAVPlus->new( redirect_path => 'cgi/securityadvisor/index.cgi' );
 
-                $self->{iav} = {
-                    installed => $store->is_product_installed(),
-                    licensed  => $store->is_product_licensed(),
-                    price     => $store->get_product_price(),
-                };
+                if ( $iavp_store->should_offer() ) {
 
-                my $iav_url = $store->get_custom_url();
-                $self->{iav}{url} = $iav_url ? $iav_url : $self->base_path('scripts13/purchase_imunifyavplus_init_SECURITYADVISOR');
+                    $self->{iavp} = {
+                        installed => $iavp_store->is_product_installed(),
+                        licensed  => $iavp_store->is_product_licensed(),
+                        price     => $iavp_store->get_product_price(),
+                    };
 
-                $self->_suggest_iav;
+                    my $iavp_url = $iavp_store->get_custom_url();
+                    $self->{iavp}{url} = $iavp_url ? $iavp_url : $self->base_path('scripts13/purchase_imunifyavplus_init_SECURITYADVISOR');
```

Comment:
This is still using scripts13 both in your update and on the "before" side. It might be helpful to base this on top of the Security Advisor change where we switched to whostmgr14 to avoid conflicts, assuming that other change is really going to go in.

Reply:
(especially since the resolution isn't just a simple matter of changing 13 to 14)

Reply:
Ah, on closer inspection, you didn't touch the part of the module that added the extra version check for the scripts14 -> scripts13 -> scripts12 fallback, so never mind. Any conflict resolution should be straightforward.

Reply:
Yeah there will certainly be merge conflicts, but shouldn't be too hard to shake out